src/util/system.h

```diff
@@ -119,7 +118,7 @@ UniValue RunCommandParseJSON(const std::string& str_command, const std::string&
  * the datadir if they are not absolute.
  *
  * @param path The path to be conditionally prefixed with datadir.
- * @param net_specific Forwarded to GetDataDir().
+ * @param net_specific Forwarded to gArgs.GetDataDirNet().
```

Comment:
This comment is not true now.
Also suggesting to move this comment update into the _"Add `ArgsManager.GetDataDirBase()` and `ArgsManager.GetDataDirNet()` as an intended replacement for `ArgsManager.GetDataDirPath(net_identifier)`"_ commit (babd5ed0204b01793c8c96d079d5304efdcc36e5).

Reply:
Right. I'm not sure about wording though. 

Maybe any of the following variants:

* `* @param net_specific Whether to append network identifier to base datadir first` or
* `* @param net_specific Use network specific datadir variant` or 
* `* @param net_specific Forwarded either to ArgsManager.GetDataDirNet() or ArgsManager.GetDataDirBase()`

Reply:
The `AbsPathForConfigVal` function should probably be made a method on `ArgsManager` anyway. The comment can be fixed up in the follow-up.

Happy to re-ACK if you push, though.

Reply:
The second one lgtm.

Reply:
> The `AbsPathForConfigVal` function should probably be made a method on `ArgsManager` anyway. The comment can be fixed up in the follow-up.

Yes, it seems so.

> The second one lgtm.

Done. Thanks.